### PR TITLE
Update 07-wallets.md

### PR DESCRIPTION
**Pull request by jnxdd**
Adding Mash as an option after testing the experience e2e as a both a sender/receiver using Mash. Some issues noticed with UI for pending HOLD invoices that we will fix.

## What does this PR do?
Adding an additional compatible wallet to the robosats list. A web-wallet/PWA so no downloads/app-stores are needed. app.mash.com/wallet. 

This PR introduces/refactors/...
n/a

### CODE REVIEW

File: docs/_pages/docs/03-understand/07-wallets.md [hunk 1]
```diff
@@ -76,6 +77,10 @@ Experience using Electrum is limited. It does not seem to support more than one
 ### LND (CLI Interface)
 Raw; it shows exactly what is happening and what it knows "IN_FLIGHT". It is not user friendly and therefore not recommended to interact with RoboSats by beginners. However, everything works just fine. If you are using LNCLI regularly, then you will find no issue using it with RoboSats.
 
+
+### Mash Wallet App (Mobile PWA & Desktop Web-Wallet)
+Overall the [Mash](https://mash.com/consumer-experience/) wallet works end2end with Robosats on both selling & receiging with all details in the notes for the invoices showin. The details of the flows in the app are shown. When the transactions are complete, they open in the mobile app on both sender/receiver sides to highlight that the transactions are completed.The one UX hick-up is that the pending invoices list doesn't explicitly show HOLD invoices (they are filtered), but will open a bug to fix this issue shortly (this note is from Aug 21st 2023).
```
Comment (Reckless-Satoshi) [inline docs/_pages/docs/03-understand/07-wallets.md:82]: Only a couple of typos, `receiging`, `showin` and a bit of dislexia `open a bug to fix this issue` :smile:

**Comment by Reckless-Satoshi** [comment]
Hey @jnxdd thank you for this PR! It's the first time I hear about Mash, what is the fastest way I can give it a try myself? :D

File: docs/_pages/docs/03-understand/07-wallets.md (hunk 1 above)
Comment (jnxdd) [reply docs/_pages/docs/03-understand/07-wallets.md]: Fixed typos.

**Comment by jnxdd** [comment]
@Reckless-Satoshi very cool. You can try the wallet out at app.mash.com/wallet (or earning with mash, although we're rebuilding and doing a big launch at app.mash.com/earn – website mash.com). If you want to protect your email and get a demo account, can share one with you. Reply here or jared@mash.com for faster.

**Comment by Reckless-Satoshi** [comment]
> If you want to protect your email and get a demo account, can share one with you. Reply here or [jared@mash.com](mailto:jared@mash.com) for faster.

Would be cool to get a demo account!

**Comment by jnxdd** [comment]
@Reckless-Satoshi have a demo account for yea... what's the best way to DM it to you so that no one jacks your sats before you do a password change!

**Comment by Reckless-Satoshi** [comment]
> demo account for yea

Amazing! Possibly Telegram will work @reckless_satoshi_v2

**Pull request merged**